### PR TITLE
Updating Azure Media Services SDK to support the 2018-07-01 API version

### PR DESCRIPTION
**Pull request by quintinb**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ x ] Please add REST spec PR link to the SDK PR
- [ x ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ one minor breaking change ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**
   - As described in the release notes, the VideoAnalyzerPreset.AudioInsightsOnly was renamed to    
     VideoAnalyzerPreset.InsightsToExtract and converted from a boolean to an enum to give three 
     options (AudioInsightsOnly, VideoInsightsOnly, and AllInsights).  The default is AllInsights.

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [ x ] Title of the pull request is clear and informative.
- [ x ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ x ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ x ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ NA ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ x ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/Media/Management.Media/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -8,7 +8,7 @@
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Media.")]
 
 [assembly: AssemblyVersion("1.0.0")]
```
Comment (dsgouda) [inline src/SDKs/Media/Management.Media/Properties/AssemblyInfo.cs]: Please update `AssemblyVersion` to `2.0.0.0`

File: src/SDKs/Media/Management.Media/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -8,7 +8,7 @@
...
-[assembly: AssemblyFileVersion("1.8.0")]
+[assembly: AssemblyFileVersion("2.0.0")]
```
Comment (dsgouda) [inline src/SDKs/Media/Management.Media/Properties/AssemblyInfo.cs]: I believe there are 4 periods in assembly versioning in `AssemblyInfo.cs`, please look at an example [here](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Cdn/Management.Cdn/Properties/AssemblyInfo.cs#L11)

**Review by dsgouda — CHANGES_REQUESTED** [review]

File: src/SDKs/Media/Management.Media/Properties/AssemblyInfo.cs (hunk 2 above)
Comment (quintinb) [reply src/SDKs/Media/Management.Media/Properties/AssemblyInfo.cs]: Changed and pushed.

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**